first-analysis-steps/code/decay-tree-fitter/ntuple_DTF1.py

```diff
@@ -24,15 +24,15 @@
 dtt.Dstar.ConsD.daughtersToConstrain = ['D0']
 
 # add another fitter, this time we will change a mass hypothesis
-dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsDpipi')
-dtt.Dstar.ConsDpipi.constrainToOriginVertex = True
-dtt.Dstar.ConsDpipi.Verbose = True
-dtt.Dstar.ConsDpipi.daughtersToConstrain = ['D0']
+dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsDKpi')
+dtt.Dstar.ConsDKpi.constrainToOriginVertex = True
+dtt.Dstar.ConsDKpi.Verbose = True
+dtt.Dstar.ConsDKpi.daughtersToConstrain = ['D0']
```

Comment:
I think this should be `['D0', 'D~0']` (and also for `dtt.Dstar.ConsD.daughtersToConstrain` a few lines above)

Reply:
For the record it's not needed, [the absolute PDG ID is constrained](https://gitlab.cern.ch/lhcb/Analysis/blob/master/Phys/DecayTreeTuple/src/TupleToolDecayTreeFitter.cpp#L671).